korge/src/commonTest/kotlin/com/soywiz/korge/ViewsForTestingTest.kt

```diff
@@ -4,31 +4,49 @@ import com.soywiz.korge.scene.Module
 import com.soywiz.korge.scene.Scene
 import com.soywiz.korge.tests.ViewsForTesting
 import com.soywiz.korge.view.*
-import com.soywiz.korim.atlas.readAtlas
 import com.soywiz.korinject.AsyncInjector
-import com.soywiz.korio.file.std.resourcesVfs
 import kotlin.test.Test
+import kotlin.test.assertEquals
 import kotlin.test.assertTrue
 
 class ViewsForTestingTest : ViewsForTesting() {
-    object DummyModule : Module() {
-        override suspend fun AsyncInjector.configure() {
-            mapPrototype {
-                DummyScene()
-            }
-        }
+    open class Dependency {
+        open val foo = "bar"
     }
 
-    class DummyScene : Scene() {
+    class DummyScene(val dependency: Dependency) : Scene() {
         var running = false
-
         override suspend fun Container.sceneInit() {
             running = true
         }
     }
 
+    object DummyModule : Module() {
+        override suspend fun AsyncInjector.configure() {
+            mapSingleton {
+                Dependency()
+            }
+            mapSingleton {
+                DummyScene(get())
```

Comment:
Scenes are intended to be prototypes instead of singletons, since everytime you change a scene a new instance is created. Even if this is a test, this could be misleading. Could you change it?

Reply:
Yeah, I'm scoping the modules so in my game they're singletons, but I definitely see your point, changing to Prototype.